### PR TITLE
gosky: bunch of moves and refactors

### PR DESCRIPTION
**Pull request by bnewbold**
Started with combining `getRepo` and `downloadRepo`, and ended up moving around a bunch of sub-commands and adding usage notes. individual commits are descriptive.

did not change any of the behavior/implementation of individual commands in this PR.

the most disruptive thing is probably moving `createInvites` to `admin createInvites`.

removed the "rebaseRepo" command entirely.

happy to revert or re-structure, this is just a proposal.

top-level output:

```
COMMANDS:
   admin          sub-commands for PDS administration
   bgs            sub-commands for administering a BGS
   car            sub-commands to work with CAR files on local disk
   createFeedGen  
   debug          a set of debugging utilities for atproto
   did            sub-commands for working with DIDs
   getRecord      
   handle         sub-commands for working handles
   list           print all of the records for a repo or local CAR file
   readStream     
   sync           sub-commands for repo sync endpoints
   help, h        Shows a list of commands or help for one command
```

### CODE REVIEW

**Comment by bnewbold** [comment]
A couple possible last things:

- split 'list' in to a version which uses the XRPC endpoints to list records, and a version which reads a local repo, and remove download-and-list functionality (unless that specific combined function is helpful? I'm not sure how gosky gets used)
- `stream` sub-command, with `readStream` renamed to `subscribe`
- `streamCompareCmd` is WIP and not accessible in current commands; could move to a separate `./cmd/diffie/` or other command, instead of being part of gosky?

File: cmd/gosky/admin.go [hunk 1]
```diff
@@ -12,21 +12,24 @@ import (
...
 	"github.com/bluesky-social/indigo/api/atproto"
+	comatproto "github.com/bluesky-social/indigo/api/atproto"
 	"github.com/bluesky-social/indigo/util/cliutil"
 	cli "github.com/urfave/cli/v2"
 )
 
 var adminCmd = &cli.Command{
-	Name: "admin",
+	Name:  "admin",
+	Usage: "sub-commands for PDS administration",
```
Comment (whyrusleeping) [inline cmd/gosky/admin.go:22]: lets move the admin-password flag here instead of being in each command

**Comment by whyrusleeping** [comment]
i'd love to have hyphenated names instead of camel case tbh.

**Comment by bnewbold** [comment]
Updates:

- did the `admin-password` refactor mentioned
- refactored camelCase to kebab-case in command names
- merged in new stuff from main, for account deletion, and moved those commands under `account`
- added a couple more usage texts

@whyrusleeping based on our convo last week I think this is ready to merge. I'd like to do it soon to avoid any more merge conflicts. Will probably merge tomorrow if there aren't any blockers.

**Pull request merged**